interface/modules/zend_modules/module/Installer/view/installer/installer/index.phtml

```diff
@@ -373,6 +373,9 @@ $help_items = [
             <th class="bg-light text-dark">
                 <span style="font-size: 1.25rem"><?php echo $listener->z_xlt('Manage Laminas Modules'); ?></span>
             </th>
+            <th style="float:right" class="bg-light text-dark">
+                <a  class="ml-3 btn btn-secondary" href="https://www.open-emr.org/wiki/index.php/OpenEMR_Modules" target="_blank"><?php echo xlt('Additional Modules'); ?></a>
+            </th>
```

Comment:
Tab is turned off so I don't understand how you are testing. I think I will delete the old tab code as looks like it causes some confusion. I left it in in case we wanted to use a tab for something else.

I like this idea and if you want I will add to my open PR and you can close this. I'll also refactor out old code.

Reply:
I can't test this because the changes that where made are not visible anywhere. I was testing in the current release and not on the bleeding edge. The bleeding edge is different as you pointed out. I see you got rid of where I thought it would go. And I just wanted to get the idea out there. Could you move it to where it would be visible? I can close this PR and you can pick it up elsewhere or take over this PR?

Reply:
Close this. I think I'll try it out either in new module help or separate button. I'll ping you and you'll be able to see it on my Gamma up for grab later.